cli/pkg/file/file_utils.go

```diff
@@ -0,0 +1,113 @@
+package file
```

Comment:
please move the most common functionalities of this file to go-utils in the appropriate package:
https://github.com/keptn/go-utils/tree/master/pkg/common 
Note, that some of them already exists. (e.g. DownloadFile)